ports/casadi/portfile.cmake

```
@@ -0,0 +1,39 @@
+vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)

```

If it is only about dllexport:
```suggestion
if(VCPKG_TARGET_IS_WINDOWS)
    vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
endif()
```